### PR TITLE
fixed TTL

### PR DESCRIPTION
**Pull request by pawelkaczmarekuk**
TTL didn't seem to work. Perhaps the issue was introduced after upgrade to spring 2.x.

I fixed the issue. All f-tests work locally.

I changed the frequency of running the ttl check to every 3s (from every 1 minute) - pls comment on that.

### CODE REVIEW

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
- functionalTests/src/functionalTest/java/uk/gov/hmcts/dm/functional/Toggles.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/document-management-store-app/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

File: functionalTests/src/functionalTest/resources/application.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+base-urls:
+    idam-user: ${IDAM_API_URL:http://localhost:4501}
```
Comment (linusnorton) [inline functionalTests/src/functionalTest/resources/application.yml]: That seems confusingly named

**Comment by linusnorton** [comment]
My only thought is that when this rolls out to prod but prod and staging prod will use documentTask-local because there is no env var set in the terraform. I think this won't cause an issue though because we don't run any functional tests against prod

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- functionalTests/src/functionalTest/java/uk/gov/hmcts/dm/functional/Toggles.java  1
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/document-management-store-app/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/document-management-store-app/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/document-management-store-app/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/document-management-store-app/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

File: functionalTests/src/functionalTest/resources/application.yml (hunk 1 above)
Comment (pawelkaczmarekuk) [reply functionalTests/src/functionalTest/resources/application.yml]: I didn't change the name of that property - just converted the file from properties to yaml

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/document-management-store-app/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
- application/src/main/java/uk/gov/hmcts/dm/config/batch/RemoveSpringBatchHistoryTasklet.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HMCTS/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- application/src/main/java/uk/gov/hmcts/dm/config/batch/RemoveSpringBatchHistoryTasklet.java  1
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HMCTS/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- application/src/main/java/uk/gov/hmcts/dm/config/batch/RemoveSpringBatchHistoryTasklet.java  1
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HMCTS/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
- application/src/main/java/uk/gov/hmcts/dm/config/batch/RemoveSpringBatchHistoryTasklet.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HMCTS/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- application/src/main/java/uk/gov/hmcts/dm/config/batch/RemoveSpringBatchHistoryTasklet.java  1
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HMCTS/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

**Comment by alectronic0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- application/src/main/java/uk/gov/hmcts/dm/config/batch/RemoveSpringBatchHistoryTasklet.java  1
- application/src/main/java/uk/gov/hmcts/dm/config/batch/DeleteExpiredDocumentsProcessor.java  1
         

Complexity decreasing per file
==============================
+ application/src/main/java/uk/gov/hmcts/dm/config/batch/BatchConfiguration.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HMCTS/document-management-store-app/pullRequest?prid=4599591&bid=15438689)

**Pull request closed without merging**